helios/pipeViewer/pipe_view/gui/autocoloring.py

```diff
@@ -1,73 +1,103 @@
+# @package Argos auto-coloring color map
 
-
-# # @package Argos auto-coloring color map
-
+from __future__ import annotations
 import wx
-import colorsys # For easy HSL to RGB conversion
-import matplotlib.pyplot as plt
-import matplotlib.patches as patches
-import warnings
+import colorsys  # For easy HSL to RGB conversion
+import numpy as np
+from typing import Dict, Set, Tuple
+
+try:
+    from daltonize import daltonize
+    HAS_DALTONIZE = True
+except ModuleNotFoundError:
+    HAS_DALTONIZE = False
```

Comment:
Imports the `daltonize` library if it's present on the system to generate colorblind-friendly palettes.